mystory/models.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+from django.utils import timezone
+
+
+class Mystory(models.Model):
```

Comment:
I think maybe the name **Story** to the model would get the point across as well and is more understandable. either way because its two words and a class is in PascalCase it should be **MyStory**.
still I think Story is better but maybe its just me 🙊

Reply:
Fixed class name